doesn't install if node-pre-gyp isn't available

```
npm i libp2p-webrtc-direct
npm ERR! code 1
npm ERR! path /Users/user/Projects/rugpulld/node_modules/wrtc
npm ERR! command failed
npm ERR! command sh -c node scripts/download-prebuilt.js
npm ERR! /bin/sh: node-pre-gyp: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user/.npm/_logs/2021-12-23T15_24_43_238Z-debug.log
```

```
npm install -g node-pre-gyp
npm WARN deprecated node-pre-gyp@0.17.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is d
eprecated and only the @mapbox scoped package will recieve updates in the future

added 67 packages, and audited 68 packages in 3s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

after installing node-pre-gyp, installing libp2p-webrtc-direct worked too

--- a/README.md
+++ b/README.md
@@ -34,6 +34,8 @@
 > npm install libp2p-webrtc-direct
 ```
 
+**NOTE:** To run build scripts `node-pre-gyp` is required. You can install it by running `npm install -g node-pre-gyp`.
+
 ## Usage
 
 ```js

